src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -210,10 +210,10 @@ public function supportsDecoding($format)
     /**
      * Flattens an array and generates keys including the path.
      */
-    private function flatten(array $array, array &$result, string $keySeparator, string $parentKey = '', bool $escapeFormulas = false)
+    private function flatten(iterable $array, array &$result, string $keySeparator, string $parentKey = '', bool $escapeFormulas = false)
```

Comment:
this is wrong imo. An iterable is not always an array (while an array is always an iterable)

Reply:
Thats the point, currently only arrays can be flatten'd but the change allows other objects to be flattend